yt_dlp/extractor/tv8.py

```diff
@@ -0,0 +1,82 @@
+from .common import InfoExtractor
+from .skyit import TV8ItIE
+from ..utils import clean_html, traverse_obj, url_or_none, urljoin
```

Comment:
`traverse_obj` should be imported from `..utils.traversal`
```suggestion
from ..utils import clean_html, url_or_none, urljoin
from ..utils.traversal import traverse_obj
```